common/etc/nginx/include/s3gateway.js

```diff
@@ -322,9 +358,9 @@ function redirectToS3(r) {
 
     if (isDirectoryListing && (r.method === 'GET' || r.method === 'HEAD')) {
         r.internalRedirect("@s3PreListing");
-    } else if ( PROVIDE_INDEX_PAGE == true ) {
+    } else if (PROVIDE_INDEX_PAGE === true) {
```

Comment:
These changes seem like they are from [this PR](https://github.com/nginxinc/nginx-s3-gateway/pull/167). Might want to remove them from or note that this PR is dependent on that one being merged first and a rebase on this one.